enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -38,20 +39,49 @@ static sgx_enclave_id_t load_enclave(bool debug_enclave)
 	int updated = 0;
 	int ret = sgx_create_enclave(ENCLAVE_FILENAME, debug_enclave, &t, &updated, &eid, NULL);
 	if (ret != SGX_SUCCESS) {
-		ETLS_ERR("Failed to load enclave %d\n", ret);
-		return -1;
+		printf("Failed to load enclave %d\n", ret);
+		return 0;
 	}
 
-	ETLS_DEBUG("Success to load enclave id %ld\n", eid);
+	printf("Success to load enclave id %ld\n", eid);
 
 	return eid;
 }
-#endif
 // clang-format on
 
-int enclave_tls_echo(int fd, enclave_tls_log_level_t log_level, char *attester_type,
-		     char *verifier_type, char *tls_type, char *crypto_type, bool mutual,
-		     bool debug_enclave)
+int enclave_tls_client_startup(enclave_tls_log_level_t log_level, char *attester_type,
+			       char *verifier_type, char *tls_type, char *crypto_type, bool mutual,
+			       bool debug_enclave, char *ip, int port)
+{
+	sgx_enclave_id_t enclave_id = 0;
+	unsigned long flags = 0;
+	uint32_t s_ip = inet_addr(ip);
+	uint16_t s_port = htons((uint16_t)port);
+
+	enclave_id = load_enclave(debug_enclave);
```

Comment:
Declare `sgx_enclave_id_t enclave_id = load_enclave(debug_encalve);` In addition, please follow the new feature "dispersion code statement" introduced by C99 for better readability.